anthos-onprem-terraform/avmw_user_cluster_metallb/main.tf

```diff
@@ -47,12 +47,26 @@ resource "google_project_service" "default" {
   disable_on_destroy = false
 }
 
+# Update platform controller with gcloud command.
+module "gcloud-update-admin-cluster-platform-controller" {
```

Comment:
```suggestion
module "gcloud_update_admin_cluster_platform_controller" {
```

Reply:
The title state `upgrade`. Should we be consistent with that and use it instead of `update`? Or is it correct to call this module `glcoud_update_...`?

Reply:
If it all should be upgrade, then please change all occurrences of update to upgrade!